_security/configuration/configuration.md

```diff
@@ -26,48 +26,64 @@ config:
       ...
 ```
 
-For a more complete example, see the [sample file on GitHub](https://github.com/opensearch-project/security/blob/main/config/config.yml).
+The sections that follow describe the main elements in each part of the `config.yml` file and provide basic examples for their configuration. For a more detailed example, see the [sample file on GitHub](https://github.com/opensearch-project/security/blob/main/config/config.yml).
 
 
 ## HTTP
 
-The `http` section has the following format:
+The `http` section includes the following format:
 
 ```yml
-anonymous_auth_enabled: <true|false>
-xff: # optional section
-  enabled: <true|false>
-  internalProxies: <string> # Regex pattern
-  remoteIpHeader: <string> # Name of the header in which to look. Typically: x-forwarded-for
-  proxiesHeader: <string>
-  trustedProxies: <string> # Regex pattern
+http:
+  anonymous_auth_enabled: <true|false>
+  xff: # optional section
+    enabled: <true|false>
+    internalProxies: <string> # Regex pattern
+    remoteIpHeader: <string> # Name of the header in which to look. Typically: x-forwarded-for
+    proxiesHeader: <string>
+    trustedProxies: <string> # Regex pattern
 ```
 
+The settings used in this configuration are described in the following table.
+
+| Setting | Description |
+| :--- | :--- |
+| `anonymous_auth_enabled` | Either enables or disables anonymous authentication. When `true`, HTTP authenticators try to find user credentials in the HTTP request. If credentials are found, the user is regularly authenticated <!--- what does "regularly authenticated" mean? --->. If none are found, the user is authenticated as an "anonymous" user. This user then has the username "anonymous" and one role named "anonymous_backendrole". When you enable anonymous authentication, all defined [HTTP authenticators](#authentication) do not challenge. See also [The challenge setting]({{site.url}}{{site.baseurl}}/security/authentication-backends/basic-authc/#the-challenge-setting). |
```

Comment:
I agree with the comment within the para. What is meant by "regularly authenticated"? Can we be more precise? Also, "are non-challenging" instead of "do not challenge"?